Jellyfin.Server/CoreAppHost.cs

```diff
@@ -78,6 +79,9 @@ protected override void RegisterServices(IServiceCollection serviceCollection)
 
             serviceCollection.AddSingleton<IActivityManager, ActivityManager>();
             serviceCollection.AddSingleton<IUserManager, UserManager>();
+            serviceCollection.AddSingleton<IAuthenticationProvider, DefaultAuthenticationProvider>();
+            serviceCollection.AddSingleton<IAuthenticationProvider, InvalidAuthProvider>();
+            serviceCollection.AddSingleton<IPasswordResetProvider, DefaultPasswordResetProvider>();
```

Comment:
Since this is moving to DI how are we going to handle external auth providers?
I would rather not move to a pattern where all plugins must register themselves

Reply:
can't plugins just overwrite the registration by using TryAddSingelton?

Reply:
After discussion, we've decided to proceed with manual service registration